### DISCUSSION TITLE
What qualifies as original?

### DISCUSSION THREAD

**Discussion by mziehm** [signalstickers/signalstickers#859] [Q&A] [answered]
I am not sure what qualifies as original? i.e. A sticker pack created based on existing graphics (which do not come from another sticker pack, but are not created by the pack author) - is this original? Only if they are modified?

**Comment by romainricard** [comment, marked as answer]
"Original" is a loose term, but I try to use it related to content. So I'd consider a pack "Original" if the actual content has, for instance, be drawn exclusively for the pack. But for a pack created from screenshots from the movie Shrek, I won't classify it as Original. If you take these screenshots, deep etch them, etc., I guess that would make the pack Original. Hope this makes sense!
